### ISSUE TITLE
JOC Bert Usage

### ISSUE THREAD

**Issue by kevinstephano** [csarofeen/pytorch#950]
## 📚 Documentation

I use these scripts to build and run my nightly jit container.  (There really isn’t any building just fetching) https://gitlab-master.nvidia.com/kstephano/mydockerfiles/-/tree/main/jit_nightly

1.) Build/Run docker container
2.) `cd /my_workspace && git clone -b kstephano/nvfuser ssh://git@gitlab-master.nvidia.com:12051/dl/JoC/bert_pyt.git joc_bert`
3.) `cd /my_workspace/joc_bert/ && pip install -r requirements.txt`
4.) Build optimizer `cd /my_workspace/joc_bert/lamb_amp_opt && python setup.py develop`
5.) Run Bert: `python3 -m torch.distributed.launch --nproc_per_node=1 /my_workspace/joc_bert/run_pretraining.py --input_dir=/data/bert/hdf5//lower_case_1_seq_len_128_max_pred_20_masked_lm_prob_0.15_random_seed_12345_dupe_factor_5_shard_1472_test_split_10/books_wiki_en_corpus/training/ --output_dir=/my_workspace/joc_bert/results/checkpoints --config_file=bert_config.json --bert_model=bert-large-uncased --train_batch_size=8192 --max_seq_length=128 --max_predictions_per_seq=20 --max_steps=100 --warmup_proportion=0.2843 --num_steps_per_checkpoint=200 --learning_rate=6e-3 --seed=42 --fp16 --gradient_accumulation_steps=128 --allreduce_post_accumulation --allreduce_post_accumulation_fp16 --do_train --skip_checkpoint`

**Comment by kevinstephano** [comment]
This documentation does not need to be an open issue. Closing...
